src/mainboard/protectli/vault_adl_n/variants/vp2430/gpio.c

```diff
@@ -0,0 +1,1415 @@
+	PAD_CFG_NF(GPP_H19, NONE, DEEP, NF1),
+
+	/* GPP_H20 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	PAD_NC(GPP_H20, NONE),
+
+	/* GPP_H21 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	PAD_NC(GPP_H21, NONE),
+
+	/* GPP_H22 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	PAD_NC(GPP_H22, NONE),
+
+	/* GPP_H23 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	PAD_NC(GPP_H23, NONE),
+
+	/* ------- GPIO Group GPP_D ------- */
+
+	/* GPP_D0 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	PAD_NC(GPP_D0, NONE),
+
+	/* GPP_D1 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	PAD_NC(GPP_D1, NONE),
+
+	/* GPP_D2 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	PAD_NC(GPP_D2, NONE),
+
+	/* GPP_D3 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	PAD_NC(GPP_D3, NONE),
+
+	/* GPP_D4 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	PAD_NC(GPP_D4, NONE),
+
+	/* GPP_D5 - SRCCLKREQ0# */
+	/* DW0: 0x44000700, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE) - IGNORED */
+	/* PAD_CFG_NF(GPP_D5, NONE, DEEP, NF1), */
+	PAD_CFG_NF(GPP_D5, NONE, DEEP, NF1),
+
+	/* GPP_D6 - SRCCLKREQ1# */
+	/* DW0: 0x44000700, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE) - IGNORED */
+	/* PAD_CFG_NF(GPP_D6, NONE, DEEP, NF1), */
+	PAD_CFG_NF(GPP_D6, NONE, DEEP, NF1),
+
+	/* GPP_D7 - SRCCLKREQ2# */
+	/* DW0: 0x44000700, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE) - IGNORED */
+	/* PAD_CFG_NF(GPP_D7, NONE, DEEP, NF1), */
+	PAD_CFG_NF(GPP_D7, NONE, DEEP, NF1),
+
+	/* GPP_D8 - SRCCLKREQ3# */
+	/* DW0: 0x44000700, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE) - IGNORED */
+	/* PAD_CFG_NF(GPP_D8, NONE, DEEP, NF1), */
+	PAD_CFG_NF(GPP_D8, NONE, DEEP, NF1),
+
+	/* GPP_D9 - BSSB_LS2_RX */
+	/* DW0: 0x44001700, DW1: 0x00003c00 */
+	/* DW0: PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE) - IGNORED */
+	/* PAD_CFG_NF(GPP_D9, NATIVE, DEEP, NF5), */
+	_PAD_CFG_STRUCT(GPP_D9, PAD_FUNC(NF5) | PAD_RESET(DEEP) | PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE), PAD_PULL(NATIVE)),
+
+	/* GPP_D10 - BSSB_LS2_TX */
+	/* DW0: 0x44001600, DW1: 0x00003c00 */
+	/* DW0: PAD_TRIG(OFF) | PAD_BUF(RX_DISABLE) - IGNORED */
+	/* PAD_CFG_NF(GPP_D10, NATIVE, DEEP, NF5), */
+	_PAD_CFG_STRUCT(GPP_D10, PAD_FUNC(NF5) | PAD_RESET(DEEP) | PAD_TRIG(OFF) | PAD_BUF(RX_DISABLE), PAD_PULL(NATIVE)),
+
+	/* GPP_D11 - BSSB_LS3_RX */
+	/* DW0: 0x44001700, DW1: 0x00003c00 */
+	/* DW0: PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE) - IGNORED */
+	/* PAD_CFG_NF(GPP_D11, NATIVE, DEEP, NF5), */
+	_PAD_CFG_STRUCT(GPP_D11, PAD_FUNC(NF5) | PAD_RESET(DEEP) | PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE), PAD_PULL(NATIVE)),
+
+	/* GPP_D12 - BSSB_LS3_TX */
+	/* DW0: 0x44001600, DW1: 0x00003c00 */
+	/* DW0: PAD_TRIG(OFF) | PAD_BUF(RX_DISABLE) - IGNORED */
+	/* PAD_CFG_NF(GPP_D12, NATIVE, DEEP, NF5), */
+	_PAD_CFG_STRUCT(GPP_D12, PAD_FUNC(NF5) | PAD_RESET(DEEP) | PAD_TRIG(OFF) | PAD_BUF(RX_DISABLE), PAD_PULL(NATIVE)),
```

Comment:
These should NC per schematics

Reply:
[NC'd](https://github.com/Dasharo/coreboot/pull/576/commits/68fe32358db1c4826321169730cb09ec89ec4a40#diff-2f9e6fbf9293a4dbeecc2314eb0b449c4eb64691bfc1c58981fc506a602441bcR354)